survey_skip_start/models/survey_survey.py

```diff
@@ -12,13 +11,7 @@ class SurveySurvey(models.Model):
         help="Skip the start screen and go directly to the survey form",
     )
 
-    @api.constrains("skip_start")
-    def _constrain_skip_start(self):
-        """For the moment, skipping the start screen is only compatible with single
-        page layout surveys"""
-        if self.filtered(lambda x: x.skip_start and x.questions_layout != "one_page"):
-            raise UserError(
-                _(
-                    "The skip start screen option is only compatible with on page layouts"
-                )
-            )
+    def _create_answer(self, *args, **additional_vals):
+        inputs = super()._create_answer(*args, **additional_vals)
```

Comment:
Please docstring about why are you overwriting this method.

Reply:
done :+1: